### PR TITLE
Release request 2.0.0

### PR DESCRIPTION
**Pull request by aleksey-kotsuba**
- Developer(s): @aleksey-kotsuba

- Ticket: https://spryker.atlassian.net/browse/ECO-2252

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Heidelpay             | major                 |                       |

#### Release Notes

* Update module to compatible with Shop Suite.
* Fix forms.
* Fix checkout plugins.
* Update license.
* Fix code style issues.

-----------------------------------------

### Improvements

* Update module to compatible with Shop Suite.
* Fix forms.
* Fix checkout plugins.
* Update license.
* Fix code style issues.

### CODE REVIEW

**Comment by dereuromark** [comment]
Please fix psr-4 namespaces

    "SprykerEcoTest\\": "tests/SprykerEcoTest/"

should be in the test namespace, not the real one.

**Comment by aleksey-kotsuba** [comment]
Thanks, fixed.

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
   },
   "autoload": {
     "psr-4": {
-      "SprykerEco\\": "src/SprykerEco/"
+      "SprykerEco\\": "src/SprykerEco/",
+      "SprykerEcoTest\\": "tests/SprykerEcoTest/"
```
[the remark this answers is not in the document]
Comment (aleksey-kotsuba) [reply composer.json]: I found nothing wrong but HeidelpayPaymentTest could not be found. I have no idea why.

[the remark this answers is not in the document]
Comment (dereuromark) [reply composer.json]: Are you running tests from project level. Thats wrong.
You must use them from the module only.

[the remark this answers is not in the document]
Comment (aleksey-kotsuba) [reply composer.json]: What do you mean? I run tests only for this module. This way works for every other modules in ECO namespace.

[the remark this answers is not in the document]
Comment (dereuromark) [reply composer.json]: Are they all exposing tests to project this way?

[the remark this answers is not in the document]
Comment (aleksey-kotsuba) [reply composer.json]: Nope. 
https://github.com/spryker-eco/adyen/blob/master/composer.json#L27
I first saw exposing tests to project in this module.

[the remark this answers is not in the document]
Comment (dereuromark) [reply composer.json]: how did u solve it?

[the remark this answers is not in the document]
Comment (aleksey-kotsuba) [reply composer.json]: There was no any problem with that. I saw this issue at first time in this module.
Anyway we will create a ticket and try to figure out the problem when we have big input to this module and have enough time foe that.

**Pull request merged**